src/lib/airtable/tables/requests.js

```diff
@@ -322,6 +322,9 @@ const fields = (exports.fields = {
   neighborhood: "Neighborhood MA-NYC",
   householdSize: "Household Size",
   forDrivingClusters: "For Driving Clusters",
+  timeline: "Timeline",
```

Comment:
Ah, I should have clarified before, these fields actually aren't 1:1 on naming. "Timeline" is actually timeSensitivity, "Needs" is determined in logic [here](https://github.com/crownheightsaid/mutual-aid-app/blob/4f56609960e8dc3e7843ace7c6818651e92bbf5a/src/slackapp/flows/createDeliveryRequest.js#L397) and "Description" is `notes`. Those are just labels applied for the Slack post. So there isn't a need to add new fields.